web/pdf_link_service.js

```diff
@@ -257,20 +257,7 @@ class PDFLinkService {
             zoomArgNumber ? zoomArgNumber / 100 : zoomArg,
           ];
         } else {
-          if (zoomArg === "Fit" || zoomArg === "FitB") {
-            dest = [null, { name: zoomArg }];
-          } else if (
-            zoomArg === "FitH" ||
-            zoomArg === "FitBH" ||
-            zoomArg === "FitV" ||
-            zoomArg === "FitBV"
-          ) {
-            dest = [
-              null,
-              { name: zoomArg },
-              zoomArgs.length > 1 ? zoomArgs[1] | 0 : null,
-            ];
-          } else if (zoomArg === "FitR") {
+          if (zoomArg === "FitR") {
```

Comment:
Why wasn't this case simply moved to the "view" branch below? It might not be in the open-parameters specification, but since we supported it previously it should remain available.

Reply:
Hi,
That is because FitR is not included in [the document](https://www.adobe.com/content/dam/acom/en/devnet/acrobat/pdfs/pdf_open_parameters_v9.pdf#page=6). So I wonder this only belongs to zoom, not view.